client/my-sites/importer/importer-blogger.jsx

```diff
@@ -48,14 +49,27 @@ class ImporterBlogger extends React.PureComponent {
 				}
 			),
 			uploadDescription: this.props.translate(
-				'Upload a {{b}}%(importerName)s export file{{/b}} to start importing into {{b}}%(siteTitle)s{{/b}}.',
+				'Upload a {{b}}%(importerName)s export file{{/b}} to start ' +
+					'importing into {{b}}%(siteTitle)s{{/b}}. If you want to know how to create the export file, ' +
+					'our {{inlineSupportLink/}} can help.',
 				{
 					args: {
 						importerName,
 						siteTitle: this.props.site.title,
 					},
 					components: {
 						b: <strong />,
+						inlineSupportLink: (
+							<InlineSupportLink
+								supportPostId={ 66764 }
```

Comment:
Out of scope for now and a little early to care for just yet, but it might be cool to have a dictionary of the articles used in Calypso some place eventually :)